components/toast.css

```diff
@@ -0,0 +1,33 @@
+.toast-success-msg {
+    padding: 0.8rem;
```

Comment:
suggestion: you can use px for values less than 1 rem

Reply:
ok